### PR TITLE
#149 Update Readme

### PR DESCRIPTION
**Pull request by shaistha24**
[#149 ](https://github.com/OpenMined/PyDP/issues/149) Create better readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,30 @@
...
 
-PyDP is part of the OpenMined community, come join the movement on [Slack](http://slack.openmined.org/).
+# PyDP
 
-# Instructions
+In today's Data-driven world, where data analytics is used by the Researchers or Data Scientists to create better models or innovative solutions for a better future! They often tend to handle sensitive or personal data, which brings in some privacy concerns. For example, sometimes, AI models can memorize details about the data they've trained on and could 'leak' these details later on. Differential privacy is a framework (using math) for measuring this leakage and reducing the possibility of it happening. It works by injecting a controlled amount of statistical noise to obscure the data contributions from individuals in the dataset. This is where PyDP comes in. PyDP is a Python wrapper for Google's [Differential Privacy](https://github.com/google/differential-privacy) project. The library provides a set of ε-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information. Thus, helping us achieve better privacy.
```
Comment (jeamick) [inline README.md]: data-driven

File: README.md [hunk 2]
```diff
@@ -1,33 +1,32 @@
-# Introduction
+![Tests](https://img.shields.io/github/workflow/status/OpenMined/PySyft/Tests)
```
Comment (codeboy5) [inline README.md]: https://img.shields.io/github/workflow/status/OpenMined/PyDP/Tests

File: README.md [hunk 3]
```diff
@@ -1,33 +1,32 @@
...
+![License](https://img.shields.io/github/license/OpenMined/PySyft)
```
Comment (codeboy5) [inline README.md]: https://img.shields.io/github/license/OpenMined/PyDP

File: README.md [hunk 4]
```diff
@@ -51,4 +51,24 @@ x.result(input_data: list)
...
-Known issue: If the privacy budget (epsilon is too less), we get a StatusOR error in the command line. While this needs to be raised as an error, right now, it's just displayed as an error in logs.
+## Learning Resources
+Some of the good learning resources to get started with Python differential privacy (PyDP) project and understand the concepts behind it can be found [here](https://github.com/OpenMined/PyDP/blob/dev/resources.md).
+
+## Join Us
+PyDP is part of the OpenMined community, come join the rapidly growing community of 7300+ on [Slack](http://slack.openmined.org/). The slack community is very friendly and great about quickly answering questions and getting your doubts cleared plus, it is a great place to interact with the fellow community member with similar interests such as yours.
+
+## Start Contributing
+
+If you'd like to contribute to this project please read these [guidelines](https://github.com/OpenMined/PyDP/blob/master/contributing.md).
```
Comment (chinmayshah99) [inline README.md]: https://github.com/OpenMined/PyDP/blob/dev/contributing.md

**Review by chinmayshah99 — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,33 +1,33 @@
...
 
-## Usage
-
-As part of the 0.1.1 dev release, we have added all functions required in carrots demo. These include: 
-- BoundedMean
-- BoundedSum
-- Max
-- Count Above
+## Installation
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install PyDP.
```
Comment (codeboy5) [inline README.md:19]: Should we mention that we only support linux and OS X right now ? @chinmayshah99

File: README.md [hunk 6]
```diff
@@ -1,33 +1,33 @@
...
 
-If you'd like to contribute to this project please read these [guidelines](https://github.com/OpenMined/PyDP/blob/master/contributing.md).
+This is where PyDP comes in. PyDP is a Python wrapper for Google's [Differential Privacy](https://github.com/google/differential-privacy) project. The library provides a set of ε-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information. Thus, helping us achieve better privacy.
 
+**Things to remember about PyDP :**
+- :rocket: Features Differentially Private Algorithmic functions as part of the 0.1.1 dev release, added to carrots demo such as BoundedMean, BoundedSum, Max, Count Above, Percentile, Min, Median , etc.  
+- All the computation methods mentioned above, use Laplace noise only. (Other noise mechanisms will be added soon... :smiley:)
+- :fire: Currently supports Linux only. (OSX, Windows coming real soon... :smiley:)
```
Comment (codeboy5) [inline README.md]: We have also added support for OS X .

File: README.md (hunk 5 above)
Comment (chinmayshah99) [reply README.md:19]: Yeah, that's a good idea! @shaistha24 please make this change.

File: README.md (hunk 6 above)
Comment (shaistha24) [reply README.md]: okey so we now support both osx and linux...

File: README.md (hunk 5 above)
Comment (shaistha24) [reply README.md:19]: ok anything else? pls look at the content one last time... may be i have missed something else?

File: README.md [hunk 7]
```diff
@@ -1,33 +1,33 @@
...
+![Version](https://img.shields.io/github/v/tag/OpenMined/PyDP?color=green&label=pypi)
+![License](https://img.shields.io/github/license/OpenMined/PyDP)
 
-PyDP is a Python wrapper for Google's [Differential Privacy](https://github.com/google/differential-privacy) project. The library provides a set of ε-differentially private algorithms, which can be used to produce aggregate statistics over numeric data sets containing private or sensitive information.
 
-PyDP is part of the OpenMined community, come join the movement on [Slack](http://slack.openmined.org/).
+# PyDP
 
-# Instructions
+In today's data-driven world, where data analytics is used by the Researchers or Data Scientists to create better models or innovative solutions for a better future! They often tend to handle sensitive or personal data, which brings in some privacy concerns. For example, sometimes, AI models can memorize details about the data they've trained on and could 'leak' these details later on. Differential privacy is a framework (using math) for measuring this leakage and reducing the possibility of it happening.
```
Comment (chinmayshah99) [inline README.md:8]: `future, they `
It's a continuation

Comment (shaistha24) [reply README.md:8]: no its the end of the line with "!"

Comment (shaistha24) [reply README.md:8]: its end of line with "!"

**Review by chinmayshah99 — APPROVED** [review]

**Comment by chinmayshah99** [comment]
Fixed #149

**Pull request merged**